ledger/participant-integration-api/src/main/scala/platform/store/dao/ParametersTable.scala

```diff
@@ -69,6 +79,11 @@ private[dao] object ParametersTable {
   def getLedgerEnd(connection: Connection): Offset =
     SelectLedgerEnd.as(LedgerEndOrBeforeBeginParser.single)(connection)
 
+  def getLedgerEndSequentialId(connection: Connection): Long =
```

Comment:
maybe there is a misunderstanding, but I thought we agreed that the mutable cache related functionality will be implemented only over append-only schema, so here I would except an OperationNotSupportedException
...let's discuss